pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -21,26 +21,54 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
 
-  patches = [ ./bin_to_env.patch ];
-
-  postPatch = ''
-    patchShebangs ./compile.sh
-    for d in scripts src/java_tools src/test src/tools third_party/ijar/test tools; do
+  patchPhase = ''
+    for d in \
```

Comment:
I'd generally prefer sticking to `postPatch` so that you can add patches if necessary and don't have to change `patchPhase` to `postPatch` again.

Is it so much slower to run `patchShebangs .` vs this whole lot? Might not be worth it and makes you need to check this each time when updating if some more shebangs need patching.